### DISCUSSION TITLE
Plugin could not load because its an array.

### DISCUSSION THREAD

**Discussion by SwordSWD** [vercel/next.js#45479] [Help]
Whenever i start my next.js. I get this error about my plugins not loading how do i fix this 
![image](https://user-images.githubusercontent.com/62721069/216051307-c3b7cbde-2005-4e09-adb2-3d95f766d7f2.png)

**Comment by icyJoseph** [comment]
Where do you see this error? Could you show us some code of where you think this is happening?

It looks like a Webpack issue, could you share your `next.config.js` or at least the relevant parts?

**Comment by SwordSWD** [comment]
[next.config.zip](https://github.com/vercel/next.js/files/10573215/next.config.zip)

**Reply by icyJoseph** [reply]
Not gonna download anything. Paste code, or share a link to a repository.

**Reply by SwordSWD** [reply]
https://pastebin.com/XFbEXZYa

**Reply by icyJoseph** [reply]
Alright one more question, what next.js version are you using?

**Reply by SwordSWD** [reply]
v12.1.16

**Reply by icyJoseph** [reply]
I still can't reproduce this:

```
    "next-banner": "^1.2.188",
    "next-compose-plugins": "^2.2.1",
    "next-images": "^1.8.4",
    "next-plugin-svgr": "^1.1.3"
```

Could you confirm which versions of these packages are you using? It looks like a problem with webpack, but I can't tell just yet.

Just so you know, Next.js error messages use colours, and we see that the error you have is uncoloured, so it must be some 3rd party dependency that's triggering this issue.
